.github/workflows/bake-aarch64.yml

```diff
@@ -92,6 +92,15 @@ jobs:
           targets: ${{ matrix.target }}
           push: ${{ inputs.no-push != true }}
           provenance: false
+      - name: Update DockerHub Description
+        if: inputs.no-push != true
+        continue-on-error: true
+        uses: peter-evans/dockerhub-description@432a30c9e07499fd01da9f8a49f0faf9e0ca5b77
+        with:
+          username: ${{ secrets.BALENAIMAGES_USER }}
+          password: ${{ secrets.BALENAIMAGES_TOKEN }}
+          repository: aarch64-alpine
+          readme-filepath: ./balena-base-images/docs/dockerhub/aarch64-alpine-full-description.md
```

Comment:
Feature creep maybe, but could add `short-description` field to update repository descriptions easily too since this is only going to run once and update the description with a deprecation notice. 

Optional.

Reply:
This will run more than once. I want to merge this change now so any images released in the next few weeks would get updated READMEs, with or without deprecation notices.

We should open a future PR to update the short description to a hardcoded string when we publish the blog post.